doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+# Offline Signing Tutorial
+
+Welcome to this tutorial on how to sign transactions offline using PSBT (Partially Signed Bitcoin Transactions). This tutorial will guide you through the process of securely signing Bitcoin transactions using an offline wallet and an online watch-only wallet.
+
+## Overview
+In this tutorial, we have two hosts: [offline] which is totally offline and without a copy of the blockchain and [online] which is a regular online node, both running Bitcoin 25.0
+
+In this example, we are going to generate a key on the offline wallet, and then send it 0.0009 BTC with an online(sending) wallet, then using the online wallet draft a transaction to send 0.0008 BTC to another online(receiving) wallet, then sign that with the offline wallet, and finally announce the signed transaction into the network using the online wallet.
+
+> [!NOTE]
+> Tested using Signet with a connected host machine and an offline docker container (representing the offline wallet) both running Signet, should also work with Regtest and Mainnet
+
+
+## Step 1: Sending money to the offline wallet
+
+### Create and Load Funds into the Online Sending Wallet
+
+1. On the online machine, create an online wallet named `online_sending_wallet` and load funds into it.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="online_sending_wallet"
+{
+  "name": "online_sending_wallet"
+}
+```
+
+2. Generate an address from the `online_sending_wallet` to receive funds.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="online_sending_wallet" getnewaddress
+tb1q5hg8nvt8frlcth57n3s2h3lfqp8he9s0ea23q4
+```
+
+3. Visit a faucet like https://signet.bc-2.jp to load funds into the generated address.
+
+
+### Create and Prepare the Offline Wallet
+
+1. On the offline machine, create an offline wallet named `offline_wallet`.
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="offline_wallet"
+{
+  "name": "offline_wallet"
+}
+```
+
+2. Export the offline wallet descriptors to a JSON file named `descriptors.json`. This file will be exported to the online machine to create a watch-only wallet.
+```sh
+[offline]$ ./src/bitcoin-cli -signet -rpcwallet="offline_wallet" listdescriptors | jq -r '.descriptors' >> ~/descriptors.json
+```
+
+### Create the online "watch_only_wallet"
+
+1. On the online machine, create an online watch-only wallet named `watch_only_wallet` without private keys.
+- The `watch_only_wallet` wallet will be used to track transactions received to the `offline_wallet` and for creating PSBT transactions.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="watch_only_wallet" disable_private_keys=true blank=true
+{
+  "name": "watch_only_wallet"
+}
+```
+
+2. Import the `offline_wallet` descriptors to the online `watch_only_wallet` using the `descriptors.json` file created on the offline wallet
```

Comment:
Might be nice to explain here that multiple descriptors are being imported and the wallets can create a variety of address types. Later when you call `getnewaddress` you can also explain the default address type and how to select another type if desired.